Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -261,7 +261,7 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var annotatesSpokenInstructions = false
     
-    var styleManager: StyleManager!
+    public private(set) var styleManager: StyleManager!
```

Comment:
jazzy is configured to hide any undocumented methods, so any new public symbol needs to be documented in order for developers to consider it available for use.